packages/cuba-react-ui/src/ui/table/DataTableHelpers.tsx

```diff
@@ -274,7 +273,7 @@ export function generateCustomFilterDropdown(
                   onOperatorChange={onOperatorChange}
                   value={value}
                   onValueChange={onValueChange}
-                  ref={customFilterRefCallback}
+                  ref={customFilterRefCallback as string & ((instance: WrappedFormUtils) => void)}
```

Comment:
nit: really hard to realize what it is, maybe move to separate type declaration?